homework_1/task_3/main.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/pkg/errors"
+	"io/ioutil"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/sethgrid/pester"
+)
+
+type pokemon struct {
+	Location_area_encounters	string
+	Name						string
+}
+
+type location struct {
+	Location_area				map[string]string
+}
+
+
+func linearBackoff(retry int) time.Duration {
+	return time.Duration(retry) * time.Second
+}
+
+func getLocations(locationURL string) ([]string, error) {
+	httpClient := pester.New()
+	httpClient.Backoff = linearBackoff
+
+	httpResponse, err := httpClient.Get(locationURL)
```

Comment:
Pester is an overkill here, you could simply use HTTP client from the standard library.

There's even a very simple `http.Get(...)` method so you don't even need to instantiate a client.
```
import "net/http"

[...]

httpResponse, err := http.Get(...)
```